src/features/applicationView/berthOfferCard/BerthOfferCard.tsx

```diff
@@ -79,7 +77,6 @@ const BerthOfferCard = ({
       }
       className={className}
       customerEmail={customerEmail}
-      invoicingDisabled={!berthInvoicingFeatureFlag()}
```

Comment:
Do we need this `invoicingDisabled` still?

Reply:
Yeah, it is used somewhere else for disabling the actions.